core/server/data/migrations/versions/5.3/2022-07-06-09-26-add-ghost-explore-integration-api-key.js

```diff
@@ -0,0 +1,69 @@
+const logging = require('@tryghost/logging');
+const security = require('@tryghost/security');
+const {default: ObjectID} = require('bson-objectid');
+const {createTransactionalMigration, meta} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Adding an admin api key for Ghost Explore Integration');
+        const integration = await knex('integrations').where({
+            slug: 'ghost-explore',
+            type: 'internal',
+            name: 'Ghost Explore'
+        }).first();
+
+        if (!integration) {
+            throw new Error('Could not find Ghost Explore Integration');
```

Comment:
I think these `Error` should be `InternalServerError`, as we try to avoid plain `Error`s in the Ghost codebase. We should really add a MigrationError at some point 🤔